scripts/zones/Lower_Jeuno/npcs/Vingijard.lua

```diff
@@ -1,20 +1,352 @@
 -----------------------------------
 -- Area: Lower Jeuno
 --  NPC: Vingijard
+--  AF Quest Reset NPC
 -----------------------------------
 local entity = {}
 
+-- keyed on xi.job
+local afQuestInfo = {
+    -- Warrior
+    [1] = {
+        questcity0 = xi.quest.log_id.BASTOK,
+        quest0 = xi.quest.id.bastok.THE_DOORMAN,
+        questcity1 = xi.quest.log_id.BASTOK,
+        quest1 = xi.quest.id.bastok.THE_TALEKEEPERS_TRUTH,
+        questcity2 = xi.quest.log_id.BASTOK,
+        quest2 = xi.quest.id.bastok.THE_TALEKEEPERS_GIFT,
+        questcity3 = xi.quest.log_id.JEUNO,
+        quest3 = xi.quest.id.jeuno.BORGHERTZ_S_WARRING_HANDS,
+    },
```

Comment:
We could make these tables a lot neater. Something like: 
```suggestion
    [xi.job.WAR] =
    {
        { xi.quest_log.id.BASTOK, xi.quest.id.bastok.THE_DOORMAN },
        { etc, etc },
    },
```

Reply:
Will have to make some amount of changes here - stylecop is complaining

Reply:
reformatted tables and changed to index access